resources/views/layouts/app.blade.php

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Kantaro</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous" />
+    <link rel="stylesheet" type="text/css" href="{{ asset('css/main.css') }}" />
```

Comment:
Are you using bootstrap? I'm using the generated (by bootstrap?) app.css and adjust it for special needs. Just wondered if there would be a preferable choice? Or do you even have an app.css? Maybe it was only created because I included bootstrap via composer.

Reply:
I do have an app.css, but it was not generated. Maybe because I do not include bootstrap via composer. I am still figuring out how to do deal with css and js in laravel